src/actions/wallets/albedoWallet.ts

```diff
@@ -0,0 +1,27 @@
+import PrivateKeyWallet from "./privateKeyWallet";
+import albedo from "@albedo-link/intent";
+
+interface IWallet {
+  publicKey: string;
+  secretKey: string;
+}
+
+interface IAlbedo {
+  destinationId: string;
+  amount: string;
+}
+
+export default class Albedo extends PrivateKeyWallet {
```

Comment:
Why does it extend from PrivateKey? It doesn't have to depend on another wallet.

Reply:
issue resolved.